src/user-guide/writing-workflows/scheduler.rst

```diff
@@ -0,0 +1,226 @@
+Scheduler Configuration
+=======================
+
+The :cylc:conf:`flow.cylc[scheduler]` section configures certain aspects of
+:term:`scheduler` behaviour at the workflow level.
+
+Many of these configurations can also be defined at the site-user level in
+the :cylc:conf:`global.cylc[scheduler]` section which is usually more
+appropriate.
+
+
+Event handlers
+--------------
+
+.. note::
+
+   Workflow event handlers are configured by:
+
+   * :cylc:conf:`flow.cylc[scheduler][events]`
+   * :cylc:conf:`global.cylc[scheduler][events]`
+
+Workflow event handlers allow configurable actions to be performed when
+workflow events occur.
+
+Workflow Events
+^^^^^^^^^^^^^^^
+
+The list of events is:
+
+startup
+   The scheduler started running the workflow.
+shutdown
+   The workflow finished and the scheduler will shut down.
+abort
+   The scheduler will shut down even though the workflow didn't finish.
+workflow timeout
+   The workflow run timed out.
+stall
+   The workflow stalled.
+stall timeout
+   The workflow timed out after stalling.
+inactivity timeout
+   The workflow timed out with no activity.
+
+You can also tell the scheduler to abort the run on certain workflow events,
+with the following settings:
+
+- abort on stall timeout
+- abort on inactivity timeout
+- abort on workflow timeout
+
+Mail Events
+^^^^^^^^^^^
+
+Cylc can send emails for workflow events, these are configured by
+:cylc:conf:`flow.cylc[scheduler][events]mail events`.
+
+For example with the following configuration, emails will be sent if a
+scheduler stalls or shuts down for an unexpected reason.
+
+.. code-block:: cylc
+
+   [scheduler]
+       [[events]]
+           mail events = stall, abort
+
+Email addresses and servers are configured by
+:cylc:conf:`global.cylc[scheduler][mail]`.
+
+Workflow event emails can be customised using
+:cylc:conf:`flow.cylc[scheduler][mail]footer`,
+:ref:`workflow_event_template_variables` can be used.
+
+For example to integrate with the Cylc 7 web interface (Cylc Review) the mail
+footer could be configured with a URL:
```

Comment:
"Integrate with it" is kind of a grandiose way to say "add a URL that points to it"?